code/game.h

```diff
@@ -37,11 +39,16 @@ bool HandleEvents(Input* input)
 	SDL_Event event;
 	bool isRunning = true;
 	InputResetMouseScroll(input);
-
 	SDL_SetRelativeMouseMode(SDL_TRUE);
 
 	uint8* keyboardState = (uint8*)SDL_GetKeyboardState(NULL);
 	InputInitKeyStates(input, keyboardState);
+	
+	for (int i = 0; i < INPUT_NUM_MOUSEBUTTONS; i++)
+	{
+		input->mouseButtonsDown[i] = 0;
+		input->mouseButtonsUp[i] = 0;	
+	}
```

Comment:
Hoe werken SDL events? Genereren die een mousedown event elk frame dat de muis down is?